chall08/sluhtala.c

```diff
@@ -0,0 +1,25 @@
+int ft_ie_except_after_c(char *str)
+{
+	int i;
+
+	i = 0;
+	if (str == NULL)
+		return (-1);
```

Comment:
Good to check the string exists. You could go all philosophical about the return values since NULL and "\0" both basically have a value 0... Buut also I don't see an empty string being a word to begin with so just having checks that prevent the program from crashing is good.